coretex/utils/process.py

```diff
@@ -40,37 +40,46 @@ class CommandException(Exception):
     pass
 
 
-def command(args: List[str], ignoreOutput: bool = False, shell: bool = False) -> None:
+def command(args: List[str], ignoreStdout: bool = False, ignoreStderr: bool = False, shell: bool = False, check: bool = True) -> Tuple[int, Optional[str], Optional[str]]:
     process = subprocess.Popen(
         args,
         shell = shell,
         cwd = Path(__file__).parent,
-        stdout = None if ignoreOutput else subprocess.PIPE,
-        stderr = None if ignoreOutput else subprocess.PIPE
+        stdout = subprocess.PIPE,
+        stderr = subprocess.PIPE
     )
 
+    stdOutStr: Optional[str] = None
+    stdErrStr: Optional[str] = None
+
     returnCode: Optional[int] = None
 
-    if ignoreOutput:
-        returnCode = process.wait()
-    else:
-        stdout = process.stdout
-        if stdout is None:
-            commandArgs = " ".join(args)
-            raise ValueError(f">> [Coretex] Something went wrong while trying to execute \"{commandArgs}\"")
+    stdout = process.stdout
+    stderr = process.stderr
+
+    if stdout is None and not ignoreStdout:
+        commandArgs = " ".join(args)
+        raise ValueError(f">> [Coretex] Something went wrong while trying to execute \"{commandArgs}\"")
 
-        while (returnCode := process.poll()) is None:
+    while (returnCode := process.poll()) is None:
+        if stdout is not None:
             lines = stdout.readlines()
-            logProcessOutput(b"".join(lines), LogSeverity.info)
+            stdOutStr = b"".join(lines).decode("utf-8")
```

Comment:
Use `stdout.readline` instead of `stdout.readlines`. This is called inside a `while` loop so we want to read line by line and log it that way. Also there is a bug here where `stdOutStr` is getting overwritten with the newly read lines, it should be appended to, not overwritten.